app/src/main/java/com/android/unio/model/association/AssociationViewModel.kt

```diff
@@ -10,25 +10,25 @@ import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.launch
 
-class ExploreViewModel(val repository: AssociationRepositoryFirestore) : ViewModel() {
+class AssociationViewModel(val repository: AssociationRepository) : ViewModel() {
```

Comment:
Very good call on using the interface as type instead of the Firestore implementation.